### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by romanowski**
This takes #7 and fixes signature rendering.

### CODE REVIEW

File: src/main/scala/dotty/dokka/tasty/TypesSupport.scala [hunk 1]
```diff
@@ -32,14 +31,22 @@ trait TypesSupport:
         case List(single) => single
         case other => other.reduce((r, e) => r ++ texts(", ") ++ e)
     
+    private def isRepeated(tpeAnnotation: Term) =
```
Comment (romanowski) [inline src/main/scala/dotty/dokka/tasty/TypesSupport.scala:34]: @AleksanderBG Any idea why this does not work? Maybe there is a better way to detect repeated parameters (varags)?

I based my logic on `RefinedPrinter` and it seems to work fine

Comment (romanowski) [reply src/main/scala/dotty/dokka/tasty/TypesSupport.scala:34]: May this bug be caused by serializaition to TASTY?

Comment (abgruszecki) [reply src/main/scala/dotty/dokka/tasty/TypesSupport.scala:34]: It seems like comparing to `defn.RepeatedParamClass` is the only way to check if something is a repeated param. I'll open a bug in Dotty and let you know when it's fixed.

**Comment by abgruszecki** [comment]
This is based on #7 and that one was reviewed, so I'm merging this one.

**Pull request merged**